packages/tools/devtools/devtools-core/src/data-visualization/SharedTreeVisualizer.ts

```diff
@@ -32,7 +82,9 @@ export function toVisualTree(tree: VisualSharedTreeNode): VisualValueNode | Visu
 		const result: VisualValueNode = {
 			value: tree.value,
 			nodeKind: VisualNodeKind.ValueNode,
-			tooltipContents: tree.schema.schemaName,
+			tooltipContents: {
```

Comment:
The docs on this function look like they could use a bit of adjustment :)

Reply:
Docs updated and added for the functions.